CONTRIBUTING.md

```diff
@@ -90,6 +90,22 @@
       `void function(const size_t /* count */) { ... }`). If the context does not provide a good name (such as
       `/* count */`), use `/* unused */`.
 
+* In general, we declare (member) variables of the same type in an own line each ([isolated declaration](https://clang.llvm.org/extra/clang-tidy/checks/readability/isolate-declaration.html)):
```

Comment:
The other guidelines are rather requests addressed to the reader:
> prefix it with `init`
> comment the parameter name out
> choose clear and concise names

So, for consistency, I would prefer formulating the new guideline in a similar way:
> Prefer declaring (member) variables of the same type in a new line each